index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Home Page</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <ul class="navbar-items">
+                <li class="logo"> <h3>Welcome</h3></li>
```

Comment:
- [ ] Kindly wrap your `<h3>Welcome</h3>` inside an `a` tag, to make it link to your home page, every logo should link to the home page.